svg/renderer.go

```diff
@@ -397,3 +365,57 @@ func (r *SVG) RenderImage(img image.Image, m canvas.Matrix) {
 	r.writeClasses(r.w)
 	fmt.Fprintf(r.w, `"/>`)
 }
+
+func writeImageBytes(img image.Image, mimetype string, w io.Writer) error {
+	if j, ok := img.(canvas.JPEGImage); ok {
+		_, err := w.Write(j.JPEGBytes())
+		return err
+	}
+	if p, ok := img.(canvas.PNGImage); ok {
+		_, err := w.Write(p.PNGBytes())
+		return err
+	}
+	if mimetype == "image/jpg" {
+		return jpeg.Encode(w, img, nil)
+	}
+	return png.Encode(w, img)
+}
+
+func (r *SVG) renderOpacityMask(img image.Image) (image.Image, string) {
```

Comment:
This function doesn't render, we should probably call it `getOpacityMask`

Reply:
It does write the mask to the renderer, line 411 (and returns the "unmasked" image)
```
fmt.Fprintf(r.w, `<mask id="%s"><image width="%d" height="%d" xlink:href="data:image/jpg;base64,`, refMask, size.X, size.Y)
```